mail/muchsync/Portfile

```diff
@@ -7,7 +7,7 @@ version             7
 categories          mail
 platforms           darwin
 license             GPL-2+
-maintainers         {seichter.de:macports @rseichter} openmaintainer
+maintainers         openmaintainer
```

Comment:
as `port lint` warns you, this is not correct if there is no maintainer listed. You should change it to `nomaintainer` in that case.

Reply:
I was wondering about that, but decided to stick to the letter of Jason Liu's mailing list response and await a review. Thank you for clearing this up.